db/hello:1:greeter_spec.rb

```diff
@@ -0,0 +1 @@
+hello/1/greeter_spec.rb
```

Comment:
I'm not sure what's going on here, but I have a feeling making this file was an accident?